packages/mdc-button/_variables.scss

```diff
@@ -15,7 +15,9 @@
 //
 
 $mdc-button-height: 36px;
+$mdc-button-horizontal-padding: 16px;
 $mdc-dense-button-height: 32px;
+$mdc-compact-button-horizontal-padding: 16px;
```

Comment:
This was `8px`, I'm assuming the change was accidental :)

Reply:
Ooops, fixed